ruddock/modules/rotation/helpers.py

```diff
@@ -0,0 +1,60 @@
+import flask
+import sqlalchemy
+
+def get_dinners():
+  query = sqlalchemy.text("""
+    SELECT DISTINCT dinner FROM rotation_prefrosh
```

Comment:
This seems inefficient. Honestly, you may want to just hardcode `DINNERS = list(range(1, 9))` as a constant.